src/test/java/bio/terra/service/tabulardata/google/BigQueryPdaoTest.java

```diff
@@ -342,6 +349,36 @@ public void snapshotBuilderQuery() throws Exception {
     List<String> searchConceptNames =
         searchConceptsResult.getResult().stream().map(SnapshotBuilderConcept::getName).toList();
     assertThat(searchConceptNames, contains("concept1"));
+
+    var rollupCountsResult =
+        snapshotBuilderService.getCountResponse(
+            dataset.getId(),
+            List.of(
+                new SnapshotBuilderCohort()
+                    .criteriaGroups(
+                        List.of(
+                            new SnapshotBuilderCriteriaGroup()
+                                .meetAll(true)
+                                .mustMeet(true)
+                                .criteria(
+                                    List.of(
+                                        new SnapshotBuilderProgramDataListCriteria()
+                                            .values(List.of(0))
+                                            .kind(SnapshotBuilderCriteria.KindEnum.LIST)
+                                            .id(1),
+                                        new SnapshotBuilderProgramDataRangeCriteria()
+                                            .high(1960)
+                                            .low(1940)
+                                            .kind(SnapshotBuilderCriteria.KindEnum.RANGE)
+                                            .id(0),
+                                        new SnapshotBuilderDomainCriteria()
+                                            .conceptId(1)
+                                            .kind(SnapshotBuilderCriteria.KindEnum.DOMAIN)
+                                            .id(19)))))),
+            TEST_USER);
+
+    // Count is 19 because of the fuzzy low counts.
+    assertThat(rollupCountsResult.getResult().getTotal(), is(19));
```

Comment:
Is this one test case (returning 19) for azure and GCP sufficient? Where did we land on the decision to have the integration and connected tests be smoke tests as compared to more thorough functional tests?

Reply:
I think we just wanted to run a test that ran the query, but didn't care about too much more than that.